content/working_with/official_plugins/Configuration/ansible.md

```diff
@@ -18,7 +18,7 @@ Execute playbook lifecycle as stand-alone node template.
 
 #### Node Operations
 
-  * `precreate`: Setup the virtual environment. Download packages in `extra_packages` and `galaxy_collections`.
```

Comment:
Let's not remove `precreate`

Reply:
do you suggest to put a disclaimer that it's valid for version <3, and >=3 we're using `configure`?

Reply:
@szpotona No. I mean we are not going to just remove the operation. It must stay in the plugin YAML and in tasks.py.